app/controllers/user/organisations_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class User::OrganisationsController < User::BaseController
+  # GET
+  def show
+    @organisation = Organisation.find(params[:id])
```

Comment:
could this be a before_filter/action that is shared with other controller methods?

Reply:
again - not a bad suggestion, but think this could be addressed in next PR